ASsets/Scripts/Level/LevelEnemySpawner.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Actor;
+using Movement;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public enum EntrySide
+{
+    Left,
+    Right
+}
+
+public class LevelEnemySpawner
+{
+    private List<EnemySpawn> spawnList = new List<EnemySpawn>();
+
+
+    public Queue<EnemySpawn> GetSpawnQueue()
+    {
+        // Spawn objects by time
+        spawnList.Sort((x, y) => x.spawnTime.CompareTo(y.spawnTime));
+        return new Queue<EnemySpawn>(spawnList);
+    }
+
+    public void CleanQueue()
+    {
+        spawnList = new List<EnemySpawn>();
+    }
+
+    public void CurvedWave(int numShips, float startTime, GameObject shipType, Vector3 startPos, EntrySide entrySide, int numRows = 1)
```

Comment:
Any reason you removed the `offset` property here? The enemies aren't all the same size (I scaled the non-aggressive enemies down to half their original size to queue the player in that they're less important), so for larger sprites, you may need more than 0.5 spacing between them.

Reply:
Sorry, could you explain what you mean? I don't see an offset property in the OG function. This was the OG header:

```c#
private void CurvedWave(int numShips, float startTime, GameObject shipType, Vector3 startPos, EntrySide entrySide, int numRows = 1)
```